Automattic/jetpack
# Connection: store persistent blog ID for multi-URL purposes

## Proposed changes:
* Store persistent blog ID in the options.
* If the current hostname is an IP address, we generate a URL secret and pass it along with other registration data.

### Other information:


## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:
1. Connect Jetpack.
2. Disconnect Jetpack.
3. Confirm the persistent blog ID is still there: `wp option get jetpack_persistent_blog_id`.
